x/claim/keeper/claim.go

```diff
@@ -209,6 +231,16 @@ func (k Keeper) GetClaimableAmountForAction(ctx sdk.Context, addr string, action
 // 	return k.distrKeeper.FundCommunityPool(ctx, sdk.NewCoins(amt), moduleAccAddr)
 // }
 
+// SetModuleAccountBalance set balance of airdrop module
+func (k Keeper) CreateModuleAccount(ctx sdk.Context, amount sdk.Coin) {
```

Comment:
here , comment should be consistent with function like
```
// CreateModuleAccount xxxxx
```

Reply:
what about the initial mint?

Reply:
Generally we don't create module account manually, as module account is not created based on seed, the address is just hash of the module name , and if you want to mint , we write the logic into `genesis.go` file. for example
```
// Mint coins into the reserve
	if data.Reserve > 0 {
		coin := common.NewCoin(common.RuneNative, cosmos.NewUint(data.Reserve))
		if err := keeper.MintToModule(ctx, ModuleName, coin); err != nil {
			panic(err)
		}
		if err := keeper.SendFromModuleToModule(ctx, ModuleName, ReserveName, common.NewCoins(coin)); err != nil {
			panic(err)
		}
	}
```